packages/editor-ui/src/Interface.ts

```diff
@@ -795,6 +795,14 @@ export interface IUserResponse {
 	personalizationAnswers?: IPersonalizationSurveyAnswers | null;
 }
 
+export interface IInviteResponse {
+	user: {
+		id: string;
+		email: string;
+	};
+	error?: string;
+}
```

Comment:
this is how it is implemented at the moment, do you want it how you did it?
```suggestion
export interface IInviteResponse {
	user: {
		id: string;
		email: string;
		error?: string;
	};
}
```